.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "dpdk"]
 	path = dpdk
 	url = ../dpdk
+[submodule "libbacktrace"]
+	path = libbacktrace
+	url = https://github.com/ianlancetaylor/libbacktrace
```

Comment:
Must we? Is there no packaged alterantive?

Note we're getting std::stacktrace() soon. https://en.cppreference.com/w/cpp/utility/basic_stacktrace

Reply:
I looked, but I didn't find any packages.

> soon

Say that again after C++20 has a fully-supporting compiler.

Reply:
```c++
$ [ -e /usr/include/c++/12/stacktrace ] && echo yay
yay
```

Reply:
> /usr/include/c++/12/stacktrace

Unsurprisingly, it's a wrapper

Reply:
> ```c++
> $ [ -e /usr/include/c++/12/stacktrace ] && echo yay
> yay
> ```

`std::stacktrace` is only available in C++23, quote from `/usr/include/c++12/stacktrace`:

> ```c++
> #if __cplusplus > 202002L && _GLIBCXX_HAVE_STACKTRACE
> ```

so, we are allowed to use C++23 features in Seastar now? or, IOW, are we allowed to use it when the tree is compiled with `-std=c++2b` ? please note, libc++ does not have it at this moment, as per https://en.cppreference.com/w/cpp/compiler_support/23 .